src/Nncase.Tests/Quant/UnitTestAssignInfoFromConfig.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NetFabric.Hyperlinq;
+using Nncase.IR;
+using Nncase.Passes;
+using Nncase.Passes.Rules.Neutral;
+using Nncase.Quantization;
+using Nncase.Tests.TestFixture;
+using Xunit;
+using static Nncase.IR.F.NN;
+
+namespace Nncase.Tests.QuantTest;
+
+[AutoSetupTestMethod(InitSession = true)]
+public class UnitTestAssignInfoFromConfig : TestClassBase
+{
+    [Fact]
+    public async Task TestAssignInfoToLeaky()
+    {
+        var input = new Var("input", new TensorType(DataTypes.Float32, new[] { 1, 3, 224, 224 }));
+        var leaky = LeakyRelu(input, 0.1);
+        var output = leaky;
+        output.Metadata.OutputNames = new string[] { "leaky_relu" };
+        var dumpVisitor = await TestAssignInfoMainPassesAsync(input, output);
+
+        Assert.Equal(0.0f, ((TensorConst)dumpVisitor.ExprMemo.Keys.ToList()[7]).Value.ToArray<float>()[0]);
+        Assert.Equal(1.0f, ((TensorConst)dumpVisitor.ExprMemo.Keys.ToList()[7]).Value.ToArray<float>()[1]);
+    }
+
+    private async Task<DumpVisitor> TestAssignInfoMainPassesAsync(Var input, Expr output)
+    {
+        CompileOptions.QuantizeOptions.ModelQuantMode = ModelQuantMode.UsePTQ;
+        CompileOptions.QuantizeOptions.QuantType = DataTypes.UInt8;
+        CompileOptions.QuantizeOptions.WQuantType = DataTypes.UInt8;
+
+        var module = new IRModule(new Function("main", output, new Var[] { input }));
+
+        var pmgr = CompileSession.CreatePassManager("Passes");
+
+        CompileOptions.QuantizeOptions.CalibrationDataset = new SolidCalibrationDatasetProvider(new Var[] { input });
+        CompileOptions.QuantizeOptions.CalibrationMethod = CalibMethod.Kld;
+        CompileOptions.QuantizeOptions.QuantScheme = File.ReadAllText(System.IO.Directory.GetCurrentDirectory() + "/../../../Quant/config_leaky.json");
```

Comment:
Emebed the json and read it using `Assembly.GetManifestResourceStream` instead of hard code path.